chevah/empirical
# [#1988] Record memory usage for tests.

# Problem description

This is initial support in empirical for getting some data for memory usage.
# Changes description

When nose-runner exists, it will print the maximum memory used by the process.

I have also created an experimental nose plugin to measure peak memory before and after test.
This is not very helpful, since we need actual memory.
# How to try and test the changes

reviewers @alibotean 
Check that code is ok.

run paver test --with-memory-usage

use latest empirical in you project.
